### PR TITLE
fix analytics example

### LINKED ISSUE #5538: amp-analytics example page broken
The [amp-analytics example](https://amp.dev/documentation/examples/components/amp-analytics/?referrer=ampbyexample.com) has a lot of AMP errors
Screen Shot 2021-03-18 at 3 21 02 PM
 is missing the TOC.

### PR DESCRIPTION
**Pull request by patrickkettner**
fixes #5538

I am assuming there is some sort of importer that would normally be running, but is failing for some reason? Additionally, the optimizer is not processing the page for some reason. Not sure why

### CODE REVIEW

File: examples/source/1.components/amp-analytics/index.html [hunk 1]
```diff
@@ -54,66 +46,6 @@
       custom-element="amp-iframe"
       src="https://cdn.ampproject.org/v0/amp-iframe-0.1.js"
     ></script>
-    <link rel="canonical" href="<% canonical %>" />
-    <meta name="viewport" content="width=device-width" />
```
Comment (sebastianbenz) [inline examples/source/1.components/amp-analytics/index.html:58]: Can we remove viewport and boilerplate if optimizer is not running on this page?

Comment (patrickkettner) [reply examples/source/1.components/amp-analytics/index.html:58]: they are redundant - sorry, I didn't include proper context in the description.

The contents of this file are being included inside of [the page](https://amp.dev/documentation/examples/components/amp-analytics/?referrer=ampbyexample.com), via [this template](https://github.com/ampproject/amp.dev/blob/future/frontend/templates/layouts/default.j2). These tags are being included on the rendered version of this page [here](https://github.com/ampproject/amp.dev/blob/future/frontend/templates/layouts/default.j2#L50) and [here](https://github.com/ampproject/amp.dev/blob/future/frontend/templates/layouts/default.j2#L73) specifically

Comment (sebastianbenz) [reply examples/source/1.components/amp-analytics/index.html:58]: What about the standalone playground versions of the examples?

Comment (patrickkettner) [reply examples/source/1.components/amp-analytics/index.html:58]: ah - I figured this was too silly to work.

Assuming things were working "correctly" - how would the duplicate tags (i.e. `<meta charset="utf-8">`) be expected to be removed?

Comment (sebastianbenz) [reply examples/source/1.components/amp-analytics/index.html:58]: The playground pulls in the original sample file.

Comment (patrickkettner) [reply examples/source/1.components/amp-analytics/index.html:58]: I understand that now, I am asking in what way is the link @CrystalOnScript provided - `https://amp.dev/documentation/examples/components/amp-analytics` expected to include that example in a way that is a valid document?

Comment (sebastianbenz) [reply examples/source/1.components/amp-analytics/index.html:58]: Let's focus on this once we have a working build and the outstanding pixie checks are done.

Comment (patrickkettner) [reply examples/source/1.components/amp-analytics/index.html:58]: sgtm

Comment (patrickkettner) [reply examples/source/1.components/amp-analytics/index.html:58]: @sebastianbenz if you would be able to answer the previous question, that would be great! thanks

File: examples/source/1.components/amp-analytics/index.html [hunk 2]
```diff
@@ -21,31 +21,23 @@
...
-      custom-element="amp-bind"
-      src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"
-    ></script>
-    <script
-      async
-      custom-element="amp-form"
-      src="https://cdn.ampproject.org/v0/amp-form-0.1.js"
-    ></script>
+    <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js" ></script>
+    <script async custom-element="amp-animation" src="https://cdn.ampproject.org/v0/amp-animation-0.1.js" ></script>
```
Comment (sebastianbenz) [inline examples/source/1.components/amp-analytics/index.html:28]: Only components should be imported that are relevant for the example. Basically: example files should be self-contained, valid AMP files.

File: examples/source/1.components/amp-analytics/index.html (hunk 1 above)
Comment (sebastianbenz) [reply examples/source/1.components/amp-analytics/index.html:58]: The components should be imported automatically via a grow extension. The boilerplate will be removed and normalized in samplesBuilder.js.

**Pull request closed without merging**